app/pods/components/otp-enroll/component.js

```diff
@@ -27,7 +27,10 @@ export default class otpEnrollComponent extends Component {
     })
     .catch(err => {
       console.error(err)
-      this.set('errorString', 'Cannot Send OTP to that email. Please use your registered email.')
+      const errorMsg = JSON.parse(err);
```

Comment:
If `err` is not a JSON string, then `JSON.parse` here will throw an error. Try using

```js
try { 
   ...
} catch (e) { 
   ... 
}

```